org.jacoco.doc/docroot/doc/resources/doc.css

```diff
@@ -1,225 +1,296 @@
-.el_jar {

```

Comment:
Seems that style of line endings was changed, which complicates review.

Reply:
@marchof I normalized line endings for this file in https://github.com/jacoco/jacoco/commit/529642a0e36965512166236fcac25ba0a20d3b97 , so could you please update this PR (merge/rebase) and resolve conflict to make your changes reviewable?